src/main/groovy/cz/alenkacz/gradle/jsonvalidator/ValidateJsonSchemaSyntaxTask.groovy

```diff
@@ -0,0 +1,44 @@
+package cz.alenkacz.gradle.jsonvalidator
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.github.fge.jackson.JacksonUtils
+import com.github.fge.jsonschema.cfg.ValidationConfiguration
+import com.github.fge.jsonschema.processors.syntax.SyntaxValidator
+import groovy.io.FileType
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.TaskAction
+
+class ValidateJsonSchemaSyntaxTask extends DefaultTask {
+    private static final ObjectMapper MAPPER = JacksonUtils.newMapper();
+    private static final def validator = new SyntaxValidator(ValidationConfiguration.byDefault())
+
+    PluginExtension pluginExtension
+
+    @TaskAction
+    def validateSchema() {
+        if (pluginExtension.schemaFolder == null) {
+            logger.info("No schema folder provided, nothing to validate")
```

Comment:
If user has not provided the schema directory/folder then it should throw exception and let the user know that he has not configured it properly. Otherwise it will be silent failure and user may not have idea why it failed.

 I am new to the Gradle so this comment may not be valid.